src/wazuh_modules/agent_upgrade/manager/wm_agent_upgrade_commands.c

```diff
@@ -45,23 +47,25 @@ static cJSON* wm_agent_upgrade_analyze_agent(int agent_id, wm_agent_task *agent_
  * @retval WM_UPGRADE_WPK_SHA1_DOES_NOT_MATCH
  * @retval WM_UPGRADE_UNKNOWN_ERROR
  * */
-static int wm_agent_upgrade_validate_agent_task(const wm_agent_task *agent_task);
+static int wm_agent_upgrade_validate_agent_task(const wm_agent_task *agent_task, wm_manager_configs manager_configs);
```

Comment:
Since `manager_configs` is used as read-only it could be defined as `const`. It is also better to use a pointer, to avoid creating a copy of the structure.

`const wm_manager_configs* manager_configs`